tests/test_requests.py

```diff
@@ -78,6 +106,17 @@ def test_request_client(scope: Scope, expected_client: Address | None) -> None:
     assert client == expected_client
 
 
+@pytest.mark.anyio
+async def test_request_close() -> None:
+    request = Request({"type": "http", "method": "GET", "path": "/foo/"})
+    assert request._form is None
+    # When request.`_form` is None, close() should do nothing.
+    # To verify this behavior, we could just call request.close() here.
+    # If request.`_form`.close is called,
+    # an Exception will be raised and the test will fail.
+    await request.close()
```

Comment:
We can add a `pragma: no branch` on the conditional branch that this is trying to cover. I don't think there's a need for this test.

Reply:
Yeah, agreed. That makes more sense.